### ISSUE TITLE
waitForViewWithAccessibilityLabel doesn't find table in childViewController

### ISSUE THREAD

**Issue by johnryan** [kif-framework/KIF#394]
I have UITableViewController as the child of another view controller. I can find this view manually but waitForViewWithAccessibilityLabel should be able to find this view (i think)

**Comment by yuriferretti-luizalabs** [comment]
@johnryan 
Did you solve this issue? I'm facing the same trouble!!

**Comment by johnryan** [comment]
I did, think I just worked around it manually, but I haven't used kif in a while

**Comment by yuriferretti-luizalabs** [comment]
@johnryan how did you find the view manually?

**Comment by johnryan** [comment]
I just recursively looked through the view hierarchy.. If you Google "find view in window" there are a bunch of stack overflow topics on how to do it

**Comment by phatmann** [comment]
Is this a change we can add to KIF?
